build/docker/lib/default_build.docker

```diff
@@ -1,8 +1,10 @@
 WORKDIR ${DIR}
 COPY . .
 
-RUN cd cmd/${CMD}; GO111MODULE=${GO111MODULE} CGO_ENABLED=0 go install "${BUILDFLAGS}"; cd - \N
-    && install -D /go/bin/${CMD} /install_root${EP}
+WORKDIR ${DIR}/cmd/${CMD}
+RUN GO111MODULE=${GO111MODULE} CGO_ENABLED=0 go install "${BUILDFLAGS}"
+WORKDIR ${DIR}
+RUN install -D /go/bin/${CMD} /install_root${EP}
```

Comment:
This would increase number of layers from one to 4. What's the benefit?

Reply:
Removal of a few hadolint warnings. But I don't think it's cost beneficial to add layers to remove a warning in this case. Thanks for noticing it!